src/shared/model/ComparisonMutationsRow.ts

```diff
@@ -0,0 +1,11 @@
+export interface ComparisonMutationsRow {
```

Comment:
I have mixed feelings about the naming. It's true that this model represents a single row in the `GroupComparisonMutationTable` which is a special summary/aggregation of multiple mutations. But conceptually it's still a mutation table. So, how about `GroupComparisonMutation`? We have a very similar concept in `SignalDB` where a single mutation actually represents multiple mutations: https://github.com/cBioPortal/cbioportal-frontend/blob/5be7d0425a0c25a3f6a5a7f895fbe948b58146eb/packages/genome-nexus-ts-api-client/src/generated/GenomeNexusAPI.ts#L750